Source/Core/Math.js

```diff
@@ -205,6 +205,7 @@ CesiumMath.FOUR_GIGABYTES = 4 * 1024 * 1024 * 1024;
  * @param {Number} value The value to return the sign of.
  * @returns {Number} The sign of value.
  */
+// eslint-disable-next-line es/no-math-sign
```

Comment:
Is there a reason we don't just want to just disable `es/no-math-sign` completely?

Reply:
This comment was put here because even the presence of `Math.sign` (despite not being called as a function) caused the linter to complain. IE doesn't support Math.sign so that's why we should keep the error everywhere else, unless there's something I don't know about.